pkg/registry/ingress.go

```diff
@@ -72,6 +72,9 @@ func GenerateIngress(cr *registryv1alpha1.DevfileRegistry, host string, scheme *
 }
 
 func GetDevfileRegistryIngress(cr *registryv1alpha1.DevfileRegistry) string {
+	if cr.Spec.K8s.IngressDomain == "" {
+		return GetHostname(cr)
```

Comment:
Not sure if we want to return just the hostname part, perhaps we return the [local hostname](https://github.com/devfile/registry-operator/blob/eff2b640984eb1bf644a8f1db11b1eb2f0c51378/pkg/registry/deployment.go#L277) as the deployed registry does not get accessed outside the cluster if the ingress is skipped. Also, add [local hostname](https://github.com/devfile/registry-operator/blob/eff2b640984eb1bf644a8f1db11b1eb2f0c51378/pkg/registry/deployment.go#L277) to [`constants.go`](https://github.com/devfile/registry-operator/blob/eff2b640984eb1bf644a8f1db11b1eb2f0c51378/pkg/registry/constants.go) as a global constant. wdyt?

This way `devfileRegistry.Status.URL` is the local address if there is no ingress created.

Reply:
I'm okay with that returning `localhost` instead of just the `GetHostname()` call! I'll make those changes.

Reply:
@Jdubrick Should also use this constant [here](https://github.com/devfile/registry-operator/blob/eff2b640984eb1bf644a8f1db11b1eb2f0c51378/pkg/registry/deployment.go#L277).